rfcs/text/pe/pe.yml

```diff
@@ -0,0 +1,193 @@
+---
+- name: pe
+  type: group
+  fields:
+    - name: main_icon
```

Comment:
`icon` v. `main_icon`?

Reply:
Not an expert, but I think you can include multiple icons for a file. The `MAINICON` is the one that is used as the actual program icon.

That said, I think it's simpler with just `icon`. If you agree, I'll make those changes.

@dcode for any input.